app/utils/ParserUtil.scala

```diff
@@ -84,12 +83,10 @@ trait ParserUtil {
     totalErrors
   }
 
-
   def getSheetErrors(schemeErrors: ListBuffer[SheetErrors]): ListBuffer[SheetErrors] = {
-    val errorCount: Int = Play.configuration.getInt(s"errorDisplayCount").getOrElse(100)
+    val errorCount: Int = appConfig.errorCount.getOrElse(HUNDRED)
```

Comment:
Should this not still be errorDisplayCount rather than errorCount?

Reply:
lazy val errorCount: Option[Int] = Option(config.getInt("errorDisplayCount"))